src/ir/named.rs

```diff
@@ -443,6 +447,9 @@ impl<'ctx, 'gen> MonotoneFramework for UsedTemplateParameters<'ctx, 'gen> {
             // parameters, but we can push the issue down the line to them.
             Some(&TypeKind::TemplateInstantiation(ref inst))
                 if !self.whitelisted_items.contains(&inst.template_definition()) => {
+                    debug!("    instantiation of blacklisted template, uses all template \
```

Comment:
Is `trace!` a lower level than `debug!`? If so, then yes I can definitely do that